src/main/scala/treadle/TreadleOptions.scala

```diff
@@ -162,6 +162,23 @@ case object RollBackBuffersAnnotation extends HasShellOptions {
   )
 }
 
+/**
+  *  Sets the number of rollback buffers in simulator, useful to see why wires have their values
```

Comment:
This comment seems to be a copy-and-paste error.

Reply:
Yep, cut and paste error. Fixed.